### PR TITLE
Javascript Functions - Michael Van Doorn

### PR DESCRIPTION
**Pull request by Michaelvxd**
CleanShot 2022-12-29 at 19 23 20

### CODE REVIEW

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Totally nothing wrong with your code but according to instructions we want to store each returned value into variable and then console log out :)

**Comment by Michaelvxd** [comment]
> Totally nothing wrong with your code but according to instructions we want to store each returned value into variable and then console log out :)

Updated!

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  var myArray = array;
+  console.log('The last element of the array is: ', myArray[array.length - 1]);
+  return myArray[array.length - 1];
+}
+
+convertMinutesToSeconds(5);
+greet('Michael');
+getArea(17, 42);
+getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+getLastElement(['propane', 'and', 'propane', 'accessories']);
```
Comment (scrumptiousdev) [inline javascript-functions/main.js:34]: Sorry for the misguidance. More like so:
```
var someFunctionResult = someFunction(something);
console.log('someFunctionResult:', someFunctionResult);
```

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**